pages/06.policies/04.trademark/docs.md

```diff
@@ -83,15 +83,14 @@ You receive an automatic license when:
 #### Examples of “fostering the Mautic platform and community”
 
 * A course entitled “How to use Mautic in your business” organized by a local non-profit organization
-* A website “mautichalloffame.com” with pictures of Mautic contributors
-* Administering a database with Mautic bugs and corresponding workarounds
+* "How-to" video(s), article(s) or blog post(s) on topics related to the Mautic platform
```

Comment:
I think that this isn't a great example to use in this section. For example, a company could claim this while they have content behind a paywall or are just using it to scrape leads or promote their Mautic services with upsells/CTAs, for example. I prefer the ones which were already provided, for this reason.